docs/operator/deploy.md

```diff
@@ -6,143 +6,114 @@ sidebarDepth: 2
 
 # Deployment guide
 
-This document explains in details the workflow to have a working group of drand
-nodes generate randomness. On a high-level, the workflow looks like this:
+This document explains the workflow to have a working group of drand nodes generate randomness. There are three main sections to this guide:
```

Comment:
"There are three main sections to this guide", but 4 bullets below

Reply:
Done!